src/Hexagon.java

```diff
@@ -5,15 +5,19 @@
     private TerrainType terrainType;
     private int tileID;
     private int level;
-    private int numVillagersOnTop;
+    private int numberOfVillagersOnTop;
```

Comment:
In an example with the boolean value, if a player puts villagers on a tile with height 2, then bool hasVillagersOnTop would be set to true, and Inventory would be updated by subtracting villagers equal to the level of the tile.  If that hex is nuked, hasVillagersOnTop would be set to false, showing that no Villagers are on top.  The number of villagers is accurately handled by an integer and the boolean*level method, it's just a matter of which do we think would be easier in the long run of this project.  It's up to you on how you want this done, I was just making a suggestion.